pkg/csi/service/wcpguest/controller.go

```diff
@@ -452,6 +467,91 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 	timeoutSeconds := int64(getAttacherTimeoutInMin(ctx) * 60)
```

Comment:
Why do we have to load the VirtualMachine CR if the `ControllerUnpublishVolume` request is for file volume?

Reply:
Removed this in the refactoring